### DISCUSSION TITLE
Using Aggregate verifiers in node-sdk

### DISCUSSION THREAD

**Discussion by ketan-jakhar** [Web3Auth/Web3Auth#1303] [Help]
Can someone help me out with how to use the aggregate verifiers in the node-sdk as boilerplate code or example is not provided for it. Also we pass idToken parameter in the connect method as well while creating aggregate verifier in array. i'm confused.

**Comment by ieow** [comment]
You can refer aggregate verifier configuration here
https://web3auth.io/docs/custom-authentication/verifiers

here is the example code 
https://github.com/Web3Auth/examples/blob/main/web-core-sdk/custom-authentication/aggregate-verifiers/auth0-google-aggregate-react-core-example/src/App.tsx

**Reply by ketan-jakhar** [reply]
But the example provided has the code to be used with REACT and not NODE-SDK on the backend
